dotnet/src/Extensions/Extensions.UnitTests/PromptTemplates/Handlebars/Helpers/KernelSystemHelpersTests.cs

```diff
@@ -57,12 +57,62 @@ public async Task ItRendersTemplateWithJsonHelperAsync()
 
         // Act
         var result = await this.RenderPromptTemplateAsync(template, arguments);
-        result = result.Replace("&quot;", "\"", StringComparison.CurrentCultureIgnoreCase);
 
         // Assert
         Assert.Equal("{\"name\":\"Alice\",\"age\":25}", result);
     }
 
+    [Fact]
+    public async Task GetHelperWithTwoArgumentsReturnsPropertyValueAsync()
+    {
+        // Arrange
+        var template = @"{{get ""person"" ""name""}}";
```

Comment:
This syntax looks odd, why use this over "person.name"

Reply:
This is just meant to catch all the cases that the model has used the `get` helper since we don't have a defined API yet. 

I'll add more restrictions once I start on this work:
https://github.com/microsoft/semantic-kernel/issues/3947 .Net: Isolate Handlebars Kernel System helpers in their own class + add Categories filter for KernelSystemHelpers (i.e., KernelHelperCategories)